lib/smart_city/dataset/business.ex

```diff
@@ -52,7 +52,7 @@ defmodule SmartCity.Dataset.Business do
             keywords: nil,
             language: nil,
             license: nil,
-            modifiedDate: nil,
+            modifiedDate: "",
```

Comment:
is this necessary if you're going to check for a nil and replace the value with an empty string? that should cover all the angles, including existing datasets that are retrieved from a view state.

Reply:
No this actually doesn't do anything valuable. removing it